buildscripts/zfs-driver/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # volume-mgmt  releases.
 #
 
-FROM ubuntu:18.04
+FROM ubuntu:19.10
```

Comment:
this looks good, but in travis we are using bionic (18.04). This might cause issues as we are compiling on 18.04 and packaging it into 19.10. https://github.com/openebs/zfs-localpv/blob/80e4d06860e9d2c3e5be0f2938c11857f15549e7/.travis.yml#L2

Reply:
Ah, I see the problem here. Travis CI doesn't even support new versions of Ubuntu yet.